AU2/frontends/inquirer_cli.py

```diff
@@ -208,11 +209,23 @@ def render(html_component, dependency_context={}):
         assassins_mapping = dependency_context[dependent]
         if not assassins_mapping:
             return {html_component.identifier: [], "skip": True}
+
+        assassins_in_event = [(aId, *rest) for (aId, *rest) in html_component.assassins if aId in assassins_mapping]
+        options = list(itertools.chain.from_iterable([
+            [(aId, i, p) for (i, p) in enumerate(pseudonyms)] for (aId, pseudonyms) in assassins_in_event]))
+
+        default_options = []
+        for (aId, idx, _) in html_component.default:
+            potential_matches = [(other_aId, p) for (other_aId, p) in assassins_in_event if other_aId == aId]
+            if potential_matches:
+                (_, p) = potential_matches[0]
+                idx_to_use = min(len(p), idx)
+                default_options.append((aId, idx_to_use, p[idx_to_use]))

```

Comment:
I don't really understand what the point of this bit is? The pseudonym list never gets shorter even if pseudonyms are 'deleted', and even if that somehow does happen the component doesn't transfer the report attributed to the now non-existent pseudonym to the 'fallback' pseudonym, so I don't really see the point of including the fallback in the default.